t/lib/Common.pm

```diff
@@ -62,12 +65,20 @@ sub skip_unless_has_secret {
     plan skip_all => 'PERL_STRIPE_TEST_API_KEY is required' unless api_key();
 }
```

Comment:
No, it wouldn't solve it. The problem is that LWP Recorder has no way to differentiate between 2 identical requests. It would only store the response for one of them. That would make mocking this test useless.